images/linux/scripts/installers/kubernetes-tools.sh

```diff
@@ -10,13 +10,8 @@ curl -L -o /usr/local/bin/kind $URL
 chmod +x /usr/local/bin/kind
 
 ## Install kubectl
-curl -s https://packages.cloud.google.com/apt/doc/apt-key.gpg | apt-key add -
-touch /etc/apt/sources.list.d/kubernetes.list
-
-# Based on https://kubernetes.io/docs/tasks/tools/install-kubectl/, package is xenial for both OS versions.
-echo "deb https://apt.kubernetes.io/ kubernetes-xenial main" | tee -a /etc/apt/sources.list.d/kubernetes.list
-apt-get update
-apt-get install -y kubectl
+curl -o /usr/local/bin/kubectl -LO https://storage.googleapis.com/kubernetes-release/release/`curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt`/bin/linux/amd64/kubectl
```

Comment:
Let's run `curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt` separately and put return value to variable. It will make code more readable